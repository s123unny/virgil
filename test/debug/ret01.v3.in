b ../core/ret01.v3 3
run
bt
info l
c
q
