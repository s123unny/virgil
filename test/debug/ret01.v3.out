breakpoint|../core/ret01.v3 3|0
stopOnBreakpoint
bt|main|../core/ret01.v3|3
variable|0|arg|0|int|false
end
result|true
